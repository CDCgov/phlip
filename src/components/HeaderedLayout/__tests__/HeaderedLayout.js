import React from 'react'
import { shallow, mount } from 'enzyme'
import { HeaderedLayout } from '../index'

const props = {
  actions: {
    logoutUser: jest.fn(),
    openMenu: jest.fn(),
    closeMenu: jest.fn()
  },
  open: false,
  menuAnchor: null
}

describe('HeaderedLayout', () => {
  test('it should render correctly', () => {
    expect(shallow(<HeaderedLayout {...props} />)).toMatchSnapshot()
  })
})