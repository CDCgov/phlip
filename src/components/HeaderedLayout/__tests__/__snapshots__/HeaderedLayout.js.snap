// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderedLayout it should render correctly 1`] = `
<WithStyles(GridWrapper)
  container={true}
  direction="column"
  spacing={0}
  style={
    Object {
      "flex": "1",
    }
  }
>
  <WithTheme(Header)
    handleCloseMenu={[Function]}
    handleLogoutUser={[Function]}
    handleOpenAdminPage={[Function]}
    open={false}
  />
  <WithStyles(GridWrapper)
    container={true}
    spacing={0}
    style={
      Object {
        "backgroundColor": "#f5f5f5",
        "flex": "1",
        "padding": "0 27px 37px 27px",
      }
    }
  />
</WithStyles(GridWrapper)>
`;
